javaslang/src/main/java/javaslang/collection/List.java

```diff
@@ -142,6 +142,8 @@
         return Nil.instance();
     }
 
+    boolean isEmpty();
```

Comment:
this was needed to unapply the default method.
TODO: this needs an `@Override` also